examples/data-transfer-mssql/main.py

```diff
@@ -70,13 +71,13 @@ def get_connection(db_host, db_name, db_username, db_password):
 def populate_data(RECORD_COUNT: int, db_cursor):
     ''' Generate user data. '''
     fake = Faker()
-    row = lambda n: (n + 1, repr(fake.name()), repr(fake.city()))
+    row: Callable[[int], Tuple[int, str, str]] = lambda n: (n + 1, repr(fake.name()), repr(fake.city()))
```

Comment:
Leave the lamdas as is. It would make it reaally verbose. But all the root level functions should be typed.

Reply:
We don't need the types here @yankeexe ☝️

Reply:
Missed that one, fixed in new commit.